.github/workflows/beta.yml

```diff
@@ -0,0 +1,116 @@
+name: Release Beta
+
+on:
+  push:
+    branches:
+      - translation-beta
+    paths-ignore:
+      - "**.md"
+
+concurrency:
```

Comment:
I would drop this block

Reply:
HI @thibault-deriv this is needed for the Workflow lists in github